src/controllers/info.ts

```diff
@@ -3,16 +3,21 @@ import { ParameterizedContext } from 'koa';
 import TMDBConfiguration from '../models/TMDBConfiguration';
 import { moviedb } from '../services/tmdb-api';
 
-export const getTMDBImageBaseURL = async(ctx: ParameterizedContext): Promise<{ imageBaseURL: string }> => {
+export const getTMDBImageBaseURL = async(): Promise<string> => {
   const configuration = await TMDBConfiguration.findOne().lean()
     .exec();
   if (configuration) {
-    return { imageBaseURL: configuration.imageBaseURL };
+    return configuration.imageBaseURL;
```

Comment:
It seems like this method will return different a different string - is that intended? What I mean is that it seems like when it is returning the `imageBaseURL` from the mongo document, it will have `'original'` appended to it. But on line 17 it will not have that appended. I dont have a good grasp on this yet so may be intended

Reply:
Good catch, will fix